Adding benefits to assessment guide (#459)
cncf/sig-security: assessments/guide/README.md

  over a week cannot be accomodated by the review team. Simply close the github
issue with a note to SIG co-chairs.

## Benefits of a Security Assessment

Having your project security assessed is fundamental for eliminating security risks.

> It allows building and maintaining a secure project over time as an integral part of the system.

Reviewer: ```suggestion
It allows one to build security as an integral part of a system and to maintain that security over time.
```
